wiktionaryparser/core.py

```diff
@@ -158,8 +158,9 @@ def parse_pronunciations(self, word_contents):
             for super_tag in list_tag.find_all('sup'):
                 super_tag.clear()
             for list_element in list_tag.find_all('li'):
-                for audio_tag in list_element.find_all('div', {'class': 'mediaContainer'}):
-                    audio_links.append(audio_tag.find('source')['src'])
+                # All audio elements are stored in the 'audiotable' class.
+                for audio_tag in list_element.find_all('table', {'class': 'audiotable'}):
+                    audio_links.append("https:" + audio_tag.find('source')['src'])
```

Comment:
```suggestion
                    audio_links.append('https:' + audio_tag.find('source')['src'])
```

The rest of the file uses `'` instead of `"`, so I imagine it's best to keep it this way.